### ISSUE TITLE
Add support for Structured data - "shippingDetails" + "hasMerchantReturnPolicy"

### ISSUE THREAD

**Issue by ShaiMagal** [PrestaShop/PrestaShop#37383]
### Prerequisites

- [x] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [x] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Structured data is very useful for Google, even indirectly it gives an overall picture of the website. We should add this structured data.

![Image](https://github.com/user-attachments/assets/70651f9b-fb37-4764-8693-1c4694bdc49e)

### Expected behavior

Support Structure data "shippingDetails" + "hasMerchantReturnPolicy"

### Steps to reproduce

1) use tool: https://search.google.com/test/rich-results
2) write any URL from clean PS
3) you will see missing structure data "shippingDetails" + "hasMerchantReturnPolicy"

### PrestaShop version(s) where the bug happened

9.0.0

### PHP version(s) where the bug happened

8.3

### Your company or customer's name goes here (if applicable).

https://www.openservis.cz/

**Comment by Hlavtox** [comment]
@ShaiMagal Yeah, google complains about this.

However, we must define how to specify this. `shippingDetails` can be automatically generated somehow, but what about `hasMerchantReturnPolicy`? We must somehow define the days that should be shown there etc.

**Comment by ShaiMagal** [comment]
@Hlavtox yes, its only warning/optional, but every structure data helping with understanding website, so i prefer to use everything possible to get the best results.

Si there is necessary to make any new section in BO for this settings.

**Comment by kpodemski** [comment]
It also complains about reviews even when the shop doesn’t use them. I think having shipping detail queried on the product page would:
- slow things down if we simulate carriers precisely
- show incorrect information otherwise 

Return policy tho is something that could be configured similar to how the ToC is configured and much easier to implement.

**Comment by ShaiMagal** [comment]
@kpodemski 
A) reviews -> Yes, because Google likes reviews ;) Check screenshot how it's working.
It's very helpfull to use reviews! ;)
![Image](https://github.com/user-attachments/assets/caa31b97-9607-435d-be7b-3fcefa2482a2)

B) shippingDetails - it's not about to choose correct delivery, we can add any settings and there you can fill range. Check documentation for shippingDetails. You will see, there isn't necessary to check any carrier, we can should give there any range.
Btw, this is from 2020. It's 2024:
https://developers.google.com/search/blog/2020/09/new-schemaorg-support-for-retailer
We should go forward, not to resist every new thing.. I understand that it's not easy, of course there are bigger priorities, but I'm simply opening this issue so that it's not forgotten in the future :) I'm not saying that we should solve it in PS9, but in PS10 it's a necessity in my opinion...
I am trying to help community with my work and my ideas ;)
